@types/interfaces.d.ts

```diff
@@ -0,0 +1,7 @@
+export type CookieConsentPosition =
+  | 'top'
+  | 'bottom'
+  | 'top-left'
+  | 'top-right'
+  | 'bottom-left'
+  | 'bottom-right';
```

Comment:
Is it intentional that this begins with `= | 'top'`?

It looks nice as they are all aligned now on separate lines, just wondering is this a good practice or not. At least TS playground doesn't seem to complain about this. Is this style used elsewhere?

Not a blocker.

Reply:
I run "format code" in VSCode which is using Prettier. But maybe it's an overkill. Reverted.